engine/commands/assignReviewer.go

```diff
@@ -21,54 +19,40 @@ func AssignReviewerCmd() *cobra.Command {
 		SilenceUsage:  true,
 		SilenceErrors: true,
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) != 1 {
+			if len(args) > 1 {
 				return fmt.Errorf("accepts 1 arg(s), received %d", len(args))
 			}
 
-			if !regexp.MustCompile(`^([a-zA-Z0-9\-]+[a-zA-Z0-9])(,[a-zA-Z0-9\-]+[a-zA-Z0-9])*$`).MatchString(args[0]) {
-				return errors.New("reviewers must be a list of comma separated valid github usernames")
+			if len(args) == 1 {
+				if _, err := strconv.ParseUint(args[0], 10, 64); err != nil {
+					return fmt.Errorf("invalid argument: %s, number of reviewers must be a number", args[0])
+				}
 			}
 
 			return nil
 		},
 		RunE: AssignReviewer,
 	}
 
-	flags := assignReviewerCmd.Flags()
-
-	flags.StringP("review-policy", "p", "reviewpad", "The policy followed for reviewer assignment. The valid values can only be: random, round-robin, reviewpad. By default, the policy is reviewpad.")
-
-	flags.Uint8P("total-reviewers", "t", 0, "The total number of reviewers to assign to. By default, it assigns to all reviewers.")
-
 	return assignReviewerCmd
 }
 
 func AssignReviewer(cmd *cobra.Command, args []string) error {
-	flags := cmd.Flags()
-
-	reviewersList := strings.Split(strings.ReplaceAll(args[0], " ", ""), ",")
-
-	availableReviewers := `"` + strings.Join(reviewersList, `","`) + `"`
-
-	totalRequiredReviewers, err := flags.GetUint8("total-reviewers")
-	if err != nil {
-		return err
+	var err error
+	totalRequiredReviewers := uint64(1)
+
+	if len(args) == 1 {
+		totalRequiredReviewers, err = strconv.ParseUint(args[0], 10, 64)
+		if err != nil {
+			return err
+		}
 	}
 
 	if totalRequiredReviewers == 0 {
-		totalRequiredReviewers = uint8(len(reviewersList))
-	}
-
-	policy, err := flags.GetString("review-policy")
-	if err != nil {
-		return err
-	}
-
-	if policy != "reviewpad" && policy != "round-robin" && policy != "random" {
-		return fmt.Errorf("invalid review policy specified: %s", policy)
+		return fmt.Errorf("invalid argument: %d, number of reviewers must be greater than 0", totalRequiredReviewers)
 	}
 
-	action := fmt.Sprintf(`$assignReviewer([%s], %d, %q)`, availableReviewers, totalRequiredReviewers, policy)
+	action := fmt.Sprintf(`$assignCodeAuthorReviewers(%d, [], 0)`, totalRequiredReviewers)
```

Comment:
No need to show the other parameters. It can be confusing.

```suggestion
	action := fmt.Sprintf(`$assignCodeAuthorReviewers(%d)`, totalRequiredReviewers)
```

Reply:
I don't believe we are transforming the programs generated by the commands with the transformer, that's why I added the defaults here but we can let the transformer take of the defaults if that's more preferable, what do you think @ferreiratiago